examples/demo-app/core/src/multiSubmit.ts

```diff
@@ -93,7 +93,7 @@ export const multiSubmit = command({
     const waitTxReceiptPromises: Promise<ContractTransactionReceipt | null>[] =
       [];
     // eslint-disable-next-line
-    const submitSolutionTxPromises: Promise<ContractTransactionReceipt | null>[] =
+    const submitSolutionReceiptPromises: Promise<ContractTransactionReceipt | null>[] =
```

Comment:
Why the null?  I can't see when that would be needed.

Reply:
It's because `txResponse.wait()` has type `Promise<null | ContractTransactionReceipt>`. We could get rid of the null if we wait for the txReceipt inside `doSubmitSolution`, but that makes the concurrency a bit less. I think that does end up being a bit nicer so I did that change.